Neos.ContentRepository.Core/Classes/Feature/NodeMove/Dto/RelationDistributionStrategy.php

```diff
@@ -41,6 +41,17 @@ public static function fromString(?string $serialization): self
             : self::STRATEGY_GATHER_ALL;
     }
 
+    public static function fromName(string $name): self
+    {
+        $name = substr($name, strpos($name, '::') + 2);
```

Comment:
can we assure here that the stuff before the "::" is alwayas the correct string "RelationDistributionStrategy"?

Reply:
done 👍